### DISCUSSION TITLE
Welcome to LearningWebGPU Discussions!

### DISCUSSION THREAD

**Discussion by hjlld** [hjlld/LearningWebGPU#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by notthistrain** [comment]
this.device = await this.adapter.requestDevice();
在开启了#enable-unsafe-webgpu的情况下，我的电脑执行requestDevice的返回值是null，不是异步编程的问题，不知是什么原因，可以帮我解答一下吗

Uncaught (in promise) TypeError: Cannot read properties of null (reading 'requestDevice')
    at App.InitGPU (app.ts?f1b2:26)
    at async eval (main.ts?794f:6)
这是报错信息

**Reply by notthistrain** [reply]
找到问题了，原来是我的远程桌面不支持3D
